radix-engine/src/blueprints/transaction_processor/executables.rs

```diff
@@ -267,6 +267,10 @@ impl<'a> ExecutableInvocation for TransactionProcessorRunInvocation<'a> {
 
         Ok((actor, call_frame_update, self))
     }
+
+    fn payload_size(&self) -> usize {
+        self.instructions.len() + self.blobs.len()
```

Comment:
Transaction size is checked by `TransacitonValidator`. Maybe we just return `0` and add a comment 

```
The payload size of TransactionProcessor invocation (allowed for ROOT only) is not limited
```

Reply:
done